### DISCUSSION TITLE
Support other TLDs

### DISCUSSION THREAD

**Discussion by Eduardogbg** [vercel/vercel#4619] [Ideas] [answered]
I would like to deploy my webapp, but I can't transfer my domain because its TLD (.com.br) isn't supported. Is supporting more TLDs planned, is there any estimative of when it might happen?

**Comment by styfle** [comment, marked as answer]
Hi @Eduardogbg 

We support many TLDs but not all as you have noted.

If you want to use an external domain, you can create a DNS record:

- `CNAME` to `cname.vercel-dns.com` for a subdomain
- `A` to `76.76.21.21` for the apex domain

https://vercel.com/docs/v2/custom-domains
